### ISSUE TITLE
Capitalize taxonomy title component using CSS rather than JS

### ISSUE THREAD

**Issue by jcklpe** [frontity/frontity#234]
I was looking over some of the additions to mars-theme and I noticed the [[title component]](https://github.com/frontity/frontity/blob/dev/packages/mars-theme/src/components/title.js) used JS to capitalize the first letter of the category names. 

```
    const { taxonomy, name } = state.source[data.taxonomy][data.id];
    // 2. Uppercase first letter of the taxonomy term (from "category" to "Category").
    const taxonomyCapitalized =
      taxonomy.charAt(0).toUpperCase() + taxonomy.slice(1);
```

I'm a designer, not a developer really so I'm unsure if this is merely ignorance on my part but this seems to be better handled [by CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform). 

Even when using css-in-js it seems best to keep as much styling in css and not do unnecessary operations like this with javascript. 

I would do a pull request for this myself it's just I'm not sure if I really am qualified to make this decision. Like I said, I'm really more a designer.

**Comment by DAreRodz** [comment]
Hey @jcklpe!

I think you are confused, the title you are talking about is the one shown in the browser bar, so the CSS there does not apply!  😅

Notice that the `<title>` is being rendered within `<Head>`, which is the Frontity component used to add elements to the `<head>` of the HTML.

**Comment by jcklpe** [comment]
@DAreRodz  Ah yes, now I see. 

I've actually integrated the `<Title>` component into my own theme but I've renamed it `<Meta>` and included the other meta/head information inside it. Title is kind of a confusing name for the component IMO, and it seems to make sense to me to just go ahead and include all the head information in one component. See an example of how I did that here: https://github.com/jcklpe/desert-jackalope/blob/master/src/components/meta.js

If you guys have any feedback on whether or not this is a good idea, let me know!
